### DISCUSSION TITLE
icqq@0.3.2启动报错

### DISCUSSION THREAD

**Discussion by gaqwe767** [Arondight/Adachi-BOT#1003] [Q&A]
![QQ截图20230508105535](https://user-images.githubusercontent.com/64371245/236728180-00735a58-d474-40bf-a43b-e28118b284d9.jpg)
icqq更新了0.3.2后，启动就报这个错误，求解决方法，谢谢。重新git clone也是报这个错误。

**Comment by Arondight** [comment]
更新下代码试试

**Reply by gaqwe767** [reply]
![QQ截图20230511112507](https://github.com/Arondight/Adachi-BOT/assets/64371245/640bdfd7-7025-408a-86bf-67d05159f4f4)
还是不能启动，提示变了

**Reply by gaqwe767** [reply]
![QQ截图20230511114136](https://github.com/Arondight/Adachi-BOT/assets/64371245/d3a9b436-cc96-4f51-97ca-388d271b13b5)
改用node app正常了，然后登录不上，换icqq@0.3.7登上又报这个错误，哎，没办法了...

**Reply by Arondight** [reply]
雀食，等上游更新吧

**Reply by Arondight** [reply]
更新一下代码应该不会报错了，现在看好像是上游还没更新协议

**Reply by Arondight** [reply]
https://github.com/icqqjs/icqq/issues/30

**Reply by gaqwe767** [reply]
今天还发现了个问题，更新到最新了，还提示上游主线分支有更新？
![QQ截图20230512184050](https://github.com/Arondight/Adachi-BOT/assets/64371245/1e65e399-3e0d-4f53-b61c-8fbfdb61d6c2)

**Reply by gaqwe767** [reply]
![QQ截图20230512185936](https://github.com/Arondight/Adachi-BOT/assets/64371245/86597dee-7b26-411d-9830-d8a72404f6db)
还有我想试试手表协议，扫码之后报错，上面的图是手表协议扫码之后报错。其他协议好像不报错，但是登录报错（禁止登录，错误45）
